src/student.rb

```diff
@@ -1,7 +1,9 @@
 require_relative 'person'
 
 class Student < Person
+  # rubocop:disable Style/OptionalBooleanParameter
   def initialize(age, classroom, name = 'Unknown', parent_permission = true)
+    # rubocop:enable Style/OptionalBooleanParameter
```

Comment:
- Please rmoeve all disabled linters, remember that linters help us to write code that is clean and easy to read, please fix this 🙏